staging/src/k8s.io/apiserver/pkg/storage/cacher.go

```diff
@@ -649,6 +649,12 @@ func (c *Cacher) isStopped() bool {
 }
 
 func (c *Cacher) Stop() {
+	// TODO : Remove the if condition below when PR
```

Comment:
nit: if clause, conditation would leave the return.

Reply:
change the verbiage a bit